src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json

```diff
@@ -0,0 +1,62 @@
+{
+  "_guid_" : "GRANT_CASE_ROLE_ASSIGNMENT",
```

Comment:
Lets prefix the file name and guid with F201_ as these files are unique to this set of tests